src/phonebook/templates/dectregistration_delete.html

```diff
@@ -0,0 +1,26 @@
+{% extends 'base.html' %}
+{% load bootstrap3 %}
+
+{% block title %}
+Delete DECT Registration | {{ block.super }}
+{% endblock %}
+
+{% block content %}
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <span class="h3">Really delete DECT registration {{ dectregistration.number }}?</span>
+  </div>
+  <div class="panel-body">
+    <p class="lead">Are you sure you want to delete the registration of DECT number {{ dectregistration.number }}?
+    <ul>
+        <li>Letters: {{ dectregistration.letters|default:"N/A" }}</li>
+        <li>Description: {{ dectregistration.description|default:"N/A" }}</li>
+    </ul>
+    <form method="post" enctype="multipart/form-data">
```

Comment:
This doesn't strictly need the `enctype="multipart/form-data"` right?

Reply:
right, copy/pasteerror :)